webpack/base.config.js

```diff
@@ -0,0 +1,39 @@
+import webpack from 'webpack';
+import yargs from 'yargs';
+
+export const argv = yargs
+  .alias('p', 'optimize-minimize')
+  .alias('d', 'debug')
+  .option('port', {
+    default: '8080',
+    type: 'string'
+  })
+  .argv;
+
+const baseConfig = {
+  entry: undefined,
+
+  output: undefined,
+
+  externals: undefined,
+
+  module: {
+    loaders: [
+      { test: /\.js/, loader: 'babel', exclude: /node_modules/ }
+    ]
+  },
+
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        'NODE_ENV': JSON.stringify(argv.optimizeMinimize ? 'production' : 'development')
+      }
+    })
+  ]
+};
+
+if (argv.optimizeMinimize) {
+  baseConfig.devtool = 'source-map';
+}
```

Comment:
That was specifically in failure stack traces when running tests in karma. I didn't try the browser though. Either way the `webpack/test.config` would override with what's needed there: https://github.com/react-bootstrap/react-bootstrap/pull/691/files#diff-0a73b4c419b2170092ad0f940a3bdc83R9